Dockerfile

```diff
@@ -63,7 +63,9 @@ RUN echo "Building OpenSSL" \
 FROM getsentry/sentry-cli:1 AS sentry-cli
 FROM relay-deps AS relay-builder
 
-ARG RELAY_FEATURES=ssl,processing
+# ssl and processing are required for basic functionality in onprem
```

Comment:
This is not exactly true, our onprem setup works well without `kafka-ssl`. I would even go as far and not build Relay with kafka-ssl as I see this as a severe edge case; however, if you are confident that there are no side effects, we can leave it on.

Reply:
yeah by the comment I mean `processing` and `ssl` are required, `kafka-ssl` is no extra cost to add